src/components/TeachersPage/LessonForm/LessonForm.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { useState } from 'react'
+
+const LessonForm: React.FC = () => {
+  const [formData, setFormData] = useState({
+    fullName: '',
+    email: '',
+    phone: '',
+    preferences: {
+      careerAndBusiness: false,
+      lessonForKids: false,
+      livingAbroad: false,
+      examsAndCoursework: false,
+      cultureTravelHobby: false
+    }
```

Comment:
```suggestion
    option: ''
```
need to do single variable (like example), because its radio buttons